roles/download-site/tasks/main.yml

```diff
@@ -27,18 +27,18 @@
 # KNOWN ISSUES:
 # --
 
-- name: Download copy of database from sites
-  shell: "drush @{{ server_alias }}.{{ site_prefix }}_{{ inventory_hostname }} sql-dump --structure-tables-key=common > /tmp/{{ inventory_hostname }}/dbdump.sql"
-  when: sites_drush_prefix == 'default'
-
 - name: Set sites drush alias
   set_fact:
-    sites_drush_alias: "{{ server_alias }}.{{ sites_drush_prefix }}{{ inventory_hostname }}"
-  when: sites_drush_prefix != 'default'
+    sites_drush_alias: '{% if sites_drush_prefix == "default" %}{{ server_alias }}.{{ site_prefix }}_{{ inventory_hostname }}{% else %}{{ server_alias }}.{{ sites_drush_prefix }}{{ inventory_hostname }}{% endif %}'
```

Comment:
We can remove the drush prefix gunk now as we determined in another PR that we wouldn't worry about adjusting this script to accommodate but rather, would like the user to adjust their aliases to accommodate.

Reply:
Eh, after working with Bill, I think we still need this.